developerportal/apps/taskqueue/celery.py

```diff
@@ -31,14 +31,21 @@
     # because we only want to run one celerybeat scheduler (for now) to keep things
     # simpler at the ops level.
     "ingest-articles-every-two-hours": {
-        "task": ("developerportal.apps.ingestion.tasks.ingest_articles"),
+        "task": "developerportal.apps.ingestion.tasks.ingest_articles",
         "schedule": crontab(minute=17, hour="*/2"),  # Every two hours, at 17 min past
         "args": (),
     },
     "ingest-videos-every-two-hours": {
-        "task": ("developerportal.apps.ingestion.tasks.ingest_videos"),
+        "task": "developerportal.apps.ingestion.tasks.ingest_videos",
         "schedule": crontab(minute=37, hour="*/2"),  # Every two hours, at 37 min past
         "args": (),
     },
+    "rebuild-search-index-weekly": {
```

Comment:
Out of curiosity: Why is this needed if the index is updated incrementally anyway? For good measure in case something falls through the cracks (which is fair)?

Reply:
Yeah, the wagtail docs recommend it but without saying why. I imagine that if there's some unexpected outage/failure during a post-save re-indexing, this is safer.